cloudmailer/cloudmailer.py

```diff
@@ -720,25 +720,57 @@ def main(argv=None):
 
     if args.schedule and not args.date:
         print ("When scheduling you need to also set a --date")
-        return 22
+        sys.exit(22)
+    # Make sure that the args date is set and have the correct format when scheduling
+    if args.schedule:
+        try:
+            datetime.strptime(args.date, '%Y-%m-%d %H:%M')
+        except ValueError:
```

Comment:
I like to get the real error and show it, so the operator can have as much information about the error as possible. So I would do: "except ValueError as error:" and then use the "error" variable in the error message attached to the explanation. In this case it might help understand what part of the date string wasn't right.

Reply:
1. It is exactly the same code as before https://github.com/CSCfi/cccp-ops-contrib/pull/21/files#diff-ba58717e0e23148385741aeda5d4a5ceb3796cc7b322770fd1042e2d82a03bc7L779-L783 

2. I don't think the operator should have more information, the operator have have themself used the `--date` and put it in worng. 

3. I will not use this functionallity of the code, I have no plan on changing it.

Reply:
It's ok then